[2024-05-06 05:07:53,284] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_Scoring_Data_Engineering_Pipeline.mapping_categorical_vars manual__2024-05-06T05:07:23.797062+00:00 [queued]>
[2024-05-06 05:07:53,288] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: Lead_Scoring_Data_Engineering_Pipeline.mapping_categorical_vars manual__2024-05-06T05:07:23.797062+00:00 [queued]>
[2024-05-06 05:07:53,288] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-05-06 05:07:53,288] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2024-05-06 05:07:53,288] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-05-06 05:07:53,309] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): mapping_categorical_vars> on 2024-05-06 05:07:23.797062+00:00
[2024-05-06 05:07:53,313] {standard_task_runner.py:52} INFO - Started process 3589 to run task
[2024-05-06 05:07:53,317] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'Lead_Scoring_Data_Engineering_Pipeline', 'mapping_categorical_vars', 'manual__2024-05-06T05:07:23.797062+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/Lead_scoring_data_pipeline/.ipynb_checkpoints/lead_scoring_data_pipeline-checkpoint.py', '--cfg-path', '/tmp/tmp57h18j_m', '--error-file', '/tmp/tmp66d_8v_x']
[2024-05-06 05:07:53,318] {standard_task_runner.py:80} INFO - Job 37: Subtask mapping_categorical_vars
[2024-05-06 05:07:53,363] {task_command.py:371} INFO - Running <TaskInstance: Lead_Scoring_Data_Engineering_Pipeline.mapping_categorical_vars manual__2024-05-06T05:07:23.797062+00:00 [running]> on host 9eefda8bbe39
[2024-05-06 05:07:53,418] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=Lead_Scoring_Data_Engineering_Pipeline
AIRFLOW_CTX_TASK_ID=mapping_categorical_vars
AIRFLOW_CTX_EXECUTION_DATE=2024-05-06T05:07:23.797062+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-05-06T05:07:23.797062+00:00
[2024-05-06 05:07:56,934] {logging_mixin.py:115} WARNING - /home/airflow/dags/Lead_scoring_data_pipeline/utils.py:185 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2024-05-06 05:07:57,257] {logging_mixin.py:115} WARNING - /home/airflow/dags/Lead_scoring_data_pipeline/utils.py:190 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2024-05-06 05:07:57,564] {logging_mixin.py:115} WARNING - /home/airflow/dags/Lead_scoring_data_pipeline/utils.py:195 SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
[2024-05-06 05:08:02,975] {python.py:173} INFO - Done. Returned value was: None
[2024-05-06 05:08:02,981] {taskinstance.py:1415} INFO - Marking task as SUCCESS. dag_id=Lead_Scoring_Data_Engineering_Pipeline, task_id=mapping_categorical_vars, execution_date=20240506T050723, start_date=20240506T050753, end_date=20240506T050802
[2024-05-06 05:08:03,018] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-05-06 05:08:03,077] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
